db/migrate/20230818114648_devise_create_users.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+class DeviseCreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      ## Database authenticatable
+      t.string :first_name,         null: false
+      t.string :last_name,          null: false
+      t.string :email,              null: false
+      t.string :encrypted_password, null: false, default: ""
```

Comment:
Is having an empty default for an encrypted password safe? I don't know exactly how devise manages new user creating but this looks a bit suspect to me